docs/AppFramework.md

```diff
@@ -7,7 +7,7 @@ The Splunk Operator provides support for Splunk App and Add-on deployment using
 Utilizing the App Framework requires:
 
 * An Amazon S3 or S3-API-compliant remote object storage location. App framework requires read-only access to the path containing the apps.
-* The remote object storage credentials.
+* The remote object storage credentials via a secret or role.
```

Comment:
As Splunk uses the term 'role' as well, this was confusing at first. Can we use "IAM role" instead?
Something like: 
* The remote object storage credentials provided as a secret, or an IAM role.
What do you think?